### DISCUSSION TITLE
投票 | 下一个版本，你想加入什么功能？

### DISCUSSION THREAD

**Discussion by lss233** [lss233/chatgpt-mirai-qq-bot#162] [Polls]
正在征集新版本功能建议，你可以在这里勾选你最想拥有的功能，我会根据投票结果优先选择功能开发。

**Comment by Hisir0909** [comment]
bingchatGPT

**Reply by mengzuihuawu** [reply]
Can you write a graphical interface, directly make a robot, and load other plug-ins, such as group management functions and the like

**Comment by Febtw** [comment]
图片OCR，包括文字图片识别，和公式图片识别并转化为 latex
